packages/fxa-graphql-api/src/lib/db/models/profile/index.ts

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Avatar } from './avatar';
+import { AvatarProvider } from './avatar-providers';
+import { Profile } from './profile';
+import { uuidTransformer } from '../../transformers';
+
+export { ProfileBaseModel } from './profile-base';
+
+export function selectedAvatar(userId: string) {
+  const uid = uuidTransformer.to(userId);
+  return Avatar.query()
```

Comment:
The code to do POST/DELETE and prerequisite code for it isn't here yet. It could be marginally simpler, though it will then need to handle retries and network failures as well.